docs/deploying_apps/deploying_node_js.md

```diff
@@ -19,8 +19,6 @@ These steps assume you have already carried out the setup process explained in t
 
 3. Include an npm ``package.json`` file to specify dependencies. The file should also specify a `start` command used to launch the app.
   
-    The presence of this file is essential to have the application detected as Node.js.
```

Comment:
@mtekel Just want to clarify about the start command: it's my understanding that you can put it in either package.json (using "start:") or manifest.yml (using command:). The original docs said to put it in package.json, because  the ability to put it in the manifest  may be removed, and they cite this comment from 2014: https://github.com/cloudfoundry/nodejs-buildpack/pull/11#issuecomment-67666273 

However, it doesn't seem like the option to put it in the manifest has been removed - so should we now recommend that people do that?